layouts/events/single.html

```diff
@@ -0,0 +1,17 @@
+{{ define "main" }}
+{{ partial "hero" . }}
+<div class="block">
+    <div class="post-content wrapper">
+        <div class="post-content__inner flow mt-6">
+            <h2>Event Details</h2>
+            <p>{{ .Params.event.summary }}</p>
+            <p class="text-black"><strong>Date:</strong> {{ dateFormat "January 2, 2006" .Params.event.date }} {{ .Params.event.start_time }} - {{ .Params.event.end_time }}</p>
+            <p class="text-black"><strong>Location:</strong> {{ .Params.event.location }}</p>
+            <p>{{ .Params.event.details }}</p>
+            {{ if .Params.event.register_link}}
+            <a href="{{ .Params.event.register_link }}" class="btn btn-primary" target="_blank" rel="noopener noreferrer">Register Now</a>
+            {{ end }}
```

Comment:
```suggestion
            {{ end }}
            {{ partial "blog/share-buttons" . }}
```

Maybe we can help them share the event by giving this some social share buttons as we have on Conway. You should be able to grab this partial and all styling right from Conway and drop it in this project.